crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -268,14 +268,14 @@ impl<'a> Visitor<'a> {
 
                     let workspace_directory = self.repo_root.resolve(workspace_info.package_path());
 
-                    let persistent = task_definition.persistent;
+                    let interactive = task_definition.interactive || task_definition.persistent;
```

Comment:
this change looks weird. persistent tasks are also considered "interactive" now?